test/unittests/test_report.py

```diff
@@ -362,3 +364,30 @@ def test_ts_to_dt():
 def test_which():
     ok_(which("ls"))
```

Comment:
I would try to remove `ok_` and use the tools coming natively in Python (or pytest).
Remeber that `nose` is deprecated and we need to go getting rid off its usage eventually